lib/reckon/app.rb

```diff
@@ -394,6 +394,15 @@ def self.parse_opts(args = ARGV)
         opts.parse!(args)
       end
 
+      if options[:file] == '-'
+        unless options[:unattended]
+          raise "--unattended is required to use STDIN as CSV source."
+        end
+
+        puts "Reading csv from STDIN"
```

Comment:
This was probably a debugging leftover?

Reply:
Good catch, I removed it, thanks!

Reply:
with https://github.com/cantino/reckon/commit/d88814f5f7f1fa479d310ceac8c4e3b7cee47313 this *looks* good to me. When I build this package pointing to that commit (I'm on nixpkgs) and run `echo "" | reckon -f - --unattended` I still get `Reading csv from STDIN`, but I'm probably just misunderstanding something on the nixpkgs side. I'll test again when 0.6.3 is out.

Reply:
can confirm this is fixed on 0.7.1!